### ISSUE TITLE
LOS bars for anaesthetists 

### ISSUE THREAD

**Issue by Sahnia** [Sahnia/Braintree-Rapid-Recovery#1]
( I think that there are too many of us going there to show anything meaningful)

**Comment by alhughes4** [comment]
agreed, could be useful if we are able to refine the number that go? I should be able to replicate what you have for the surgeons but change the group_by

**Comment by Sahnia** [comment]
I have added this. 

It has got me thinking if we should sort bars by the number  of cases

**Comment by alhughes4** [comment]
Would make it aesthetically more pleasing?

**Comment by Sahnia** [comment]
I wonder if it will highlight any association between case numbers and LOS.

**Comment by alhughes4** [comment]
It would be logically to think that there may be... joys of having data to look at over a longitudinal time frame.
